lib/ansible/module_utils/netapp.py

```diff
@@ -48,12 +51,6 @@
 except ImportError:
     HAS_NETAPP_LIB = False
 
-try:
```

Comment:
It seems your version of netapp.py is not synced with the latest changes we submitted to GItHub.

Reply:
Are they pulled in? I did a fetch and rebased before I pushed my changes.